### DISCUSSION TITLE
HTTP2

### DISCUSSION THREAD

**Discussion by johnvo9595** [serversideup/docker-php#211] [Q&A] [answered]
How i can use http2 with php fpm nginx??

**Comment by jaydrogers** [comment, marked as answer]
It's enabled by default 🤓:
https://github.com/serversideup/docker-php/blob/main/src/fpm-nginx/etc/nginx/site-opts.d/https.conf#L1

Just be sure to set your SSL_MODE to `full`:
https://serversideup.net/open-source/docker-php/docs/reference/environment-variable-specification
